kolibri/plugins/coach/kolibri_plugin.py

```diff
@@ -59,19 +57,3 @@ def plugin_data(self):
         return {
             "practice_quizzes_exist": practice_quizzes_exist,
         }
-
-
-@register_hook
```

Comment:
This is just deferring all cleanup to the garbage collection, right?

Reply:
Yes, which I made the PR description say this depends on that PR